openlcs/libs/unpack.py

```diff
@@ -75,21 +89,18 @@ def unpack_file(file_name, main_dir=None):
         abs_path = os.path.abspath(file_name)
         file_name = os.path.basename(file_name)
         file_path = main_dir or os.path.dirname(abs_path)
-        cmd = ("file -i -b -L '%s' | grep -q 'text'" % file_name)
-        retcode, _ = run(cmd, workdir=file_path, can_fail=True)
-        if retcode == 0:
-            return None
-
-        tmp_dir = tempfile.mkdtemp(prefix='tmpunpack_', dir=main_dir)
-        cmd = ("atool -X '%(tmp_dir)s' -q '%(file_path)s/%(file_name)s' "
-               ">/dev/null" % locals())
-        try:
-            run(cmd, workdir=file_path)
-            return tmp_dir
-        except RuntimeError:
-            shutil.rmtree(tmp_dir)
-            err_msg = 'Unable to decompress file %s.' % file_name
-            raise ValueError(err_msg) from None
+        _, file_extension = os.path.splitext(file_name)
```

Comment:
```
>>> file_name = "test.tar.gz"
>>> import os
>>> _, file_extension = os.path.splitext(file_name)
>>> file_extension
'.gz'
>>> 
```

Reply:
I think use the following method is better:
```
import pathlib
file_name = "test.tar.gz"
file_extension = ''.join(pathlib.Path(file_name).suffixes)
print(file_extension)
```

Reply:
Good catch! Updated.